### PR TITLE
Add in sprocket blog post.

### PR DESCRIPTION
**Pull request by vsmalladi**

### CODE REVIEW

File: _posts/2024-9-20-announcing-sprocket [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+title: "Sprocket"
+author: Venkat Malladi
+date:   2024-09-20 15:24:15 -0600
+categories: wdl bioinformatics workflows vscode lint
+---
+
+# Introducing Sprocket
+
+We are excited to announce **Sprocket**, a linter and validity of Workflow Description Language (WDL). Additionally, it is underactive development to become a workflow engine for bioinformatics and computational research. Sprocket is designed to be highly compatible with WDL, making it a valuable addition to the growing ecosystem of WDL tools.
```
Comment (claymcleod) [inline _posts/2024-9-20-announcing-sprocket:10]:
```suggestion
We are excited to announce **Sprocket**, a validator and linter for the Workflow Description Language (WDL). Beyond just validation and linting, Sprocket is under active development towards becoming a workflow execution engine for bioinformatics and computational research. Sprocket uses WDL as it's workflow language of choice, making it a valuable addition to the growing ecosystem of WDL tools.
```

File: _posts/2024-9-20-announcing-sprocket [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+
+## What is Sprocket?
+
+
+Sprocket is written in Rust. The code that drives Sprocket is split across the [wdl](https://github.com/stjude-rust-labs/wdl) family of crates, the [sprocket](https://github.com/stjude-rust-labs/sprocket) command line tool, and the [Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=stjude-rust-labs.sprocket-vscode).
```
Comment (claymcleod) [inline _posts/2024-9-20-announcing-sprocket:16]:
```suggestion
Sprocket is written in Rust. The code that drives Sprocket is split across the [`wdl`](https://github.com/stjude-rust-labs/wdl) family of crates, the [`sprocket`](https://github.com/stjude-rust-labs/sprocket) command line tool, and the [Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=stjude-rust-labs.sprocket-vscode).
```

File: _posts/2024-9-20-announcing-sprocket [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+- **WDL Compatibility**: Full support for workflows written in WDL, allowing users to define complex workflows in a readable and intuitive format.
+- **Lightweight and Fast**: Optimized for speed and minimal resource usage, Sprocket is a perfect fit for small-to-medium-sized computational projects.
+- **Flexible Deployment**: Sprocket can run on local machines, clusters, and cloud environments, giving users the freedom to scale their workflows as needed.
+- **Open-Source**: Developed under an open-source license, Sprocket encourages contributions from the community to extend its capabilities.
+
+## Why Use Sprocket?
+
+Sprocket aims at providing a minimal yet robust alternative for testing and running WDL workflows.
+
+- **Simplicity**: For users who need to run WDL workflows quickly without complex infrastructure setup, Sprocket offers a straightforward solution.
```
Comment (claymcleod) [inline _posts/2024-9-20-announcing-sprocket:29]: This is a pretty interesting point to call out—we're specifically targeting mid to high scale bioinformatics workflow analysis. In particular, I do think there will be other solutions that are better for very small WDL analysis (e.g., miniwdl).

File: _posts/2024-9-20-announcing-sprocket [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+
+## Getting Started with Sprocket
+
+Interested in trying out Sprocket for your next project? Head over to the [official Sprocket documentation](https://stjude-rust-labs.github.io/sprocket/overview.html) for detailed installation instructions and usage guides. Whether you are working on local workflows or deploying large-scale computations, Sprocket's flexibility ensures you can adapt it to fit your needs.
+
+
+## What's Next?
+
+Sprocket is under active development, and we welcome contributions from the community! Whether you want to suggest new features, report bugs, or contribute code, check out our [GitHub repository](https://github.com/stjude-rust-labs/sprocket) and join the conversation.
```
Comment (claymcleod) [inline _posts/2024-9-20-announcing-sprocket:63]: Perhaps also a call to join the Sprocket channel on Slack?

Comment (vsmalladi) [reply _posts/2024-9-20-announcing-sprocket:63]: Good point will add that now

**Pull request merged**